src/ast/seq_decl_plugin.h

```diff
@@ -273,6 +273,21 @@ class seq_util {
         bool is_in_re(expr const* n)    const { return is_app_of(n, m_fid, OP_SEQ_IN_RE); }
         bool is_unit(expr const* n)     const { return is_app_of(n, m_fid, OP_SEQ_UNIT); }
 
+        bool is_string_term(expr const * n) const {
```

Comment:
not sure how _STRING_SORT gets into play. The seq_decl_plugin converts this to a sequence over bit-vectors.